core/common/src/main/java/alluxio/collections/IndexedSet.java

```diff
@@ -128,21 +138,83 @@
   }
 
   /**
+   * An interface extending{@link FieldIndex}, represents a unique index.
+   *
+   * @param <T> type of objects in this {@link IndexedSet}
+   */
+  public interface UniqueFieldIndex<T> extends FieldIndex<T> {}
+
+  /**
+   * An interface extending{@link FieldIndex}, represents a nonunique index.
+   *
+   * @param <T> type of objects in this {@link IndexedSet}
+   */
+  public interface NonUniqueFieldIndex<T> extends FieldIndex<T> {}
```

Comment:
it seems that no reference in production code (non-testing code) for this `NonUniqueFieldIndex` after this PR , is it correct?